regression/rt_iso_euler_sodshock_dg_comparison.c

```diff
@@ -1,128 +1,309 @@
+// Sod-type shock tube test, with a small jump in state variables for comparison against Discontinuous Galerkin methods, for the isothermal Euler equations.
```

Comment:
You can delete this regression test. I forgot we have it. I had some breakthroughs in November for DG fluids that allow us to do the regular sod shock using a characteristic based limiter and I don't see a reason to support two isothermal Euler sod shocks with different left and right states.

Reply:
Roger roger.